index.js

```diff
@@ -248,16 +246,9 @@ Dat.prototype.download = function (cb) {
     self.emit('upload', data)
   })
 
-  function updateTotalStats () {
-    self.stats.bytesTotal = archive.content ? archive.content.bytes : 0
-    var fileCount = 0
-    each(archive.list({live: false}), function (data, next) {
-      if (data.type === 'file') fileCount++
-      next()
-    }, function () {
-      self.stats.filesTotal = fileCount
-    })
-  }
+  self._stats.on('update', function () {
+    self.stats = extend(self.stats, self._stats.get())
```

Comment:
For this module it might actually make a lot more sense to have hem in hyperdrive-stats, so

1) they're persistent
2) you don't need 2 index instances